esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -404,6 +406,7 @@ define([
                         context.thorProcess.set("value", targetData[0].ClusterGroup);
                 } else {
                    context.slaveLogs.set("disabled", true);
+                   context.includeSlaveLogsCheckbox.set("disabled", true);
```

Comment:
The checkbox is to include the slave logs in the ZAP report - the other is complete drop down menu option.